arcade/draw_commands.py

```diff
@@ -617,11 +618,16 @@ def draw_triangle_outline(x1: float, y1: float,
 # --- BEGIN RECTANGLE FUNCTIONS # # #
 
 
+@warning(
+    message=\
+        "Drawing with lrtb(left, right, top, bottom) is deprecated use lrbt instead i.e draw_lrbt_rectangle_outline.",
+    warning_type=NameChangeWarning,
+)
 def draw_lrtb_rectangle_outline(left: float, right: float, top: float,
```

Comment:
Maybe make the message a bit shorter?
`draw_lrtb_rectangle_outline is deprecated. Use draw_lrbt_rectangle_outline instead.`

Reply:
Updated in https://github.com/pythonarcade/arcade/commit/f421cb9a1ed17d82a3c817eaac4fb60fca2a223e